wagtail/contrib/redirects/tests/test_import_utils.py

```diff
@@ -45,6 +47,19 @@ def test_that_cache_storage_are_returned(self):
         FileStorage = get_file_storage()
         self.assertEqual(FileStorage.__name__, "RedirectsCacheStorage")
 
+    @override_settings(WAGTAIL_REDIRECTS_FILE_STORAGE="cache")
+    def test_cache_storage_remove_uses_prefix(self):
+        storage = CacheStorage(name='test_key')
+        storage.save('test_content')
+        self.assertEqual(storage.read(), 'test_content')
+
+        storage.remove()
+        self.assertIsNone(storage.read())
+
+        # Verify that the cache key includes the prefix explicitly
+        prefix = getattr(storage, 'CACHE_PREFIX', '')
```

Comment:
Nit: This attribute is always defined
```suggestion
        prefix = storage.CACHE_PREFIX
```